SP23.P03.Web/ClientApp/src/pages/TrainStations/StationListingPage.css

```diff
@@ -0,0 +1,47 @@
+
+.modal-content {
+    display: flex;
+    max-width: fit-content;
+    padding: 2em;
+}
+
+.center-form {
+    text-align: center;
+}
+
+.ui.container.stations {
+    margin-top: 2em;
+    border: 2px solid var(--main-color) !important;
+}
+
+.ui.modal.create {
+    max-width: 30%;
+    padding: 2em;
+    border: 2px outset var(--main-color) !important;
+    text-align: center;
+}
+
+/*
+.form-group > label {
+    float: left;
+}
+*/
+
+h2 {
+    padding-bottom: 1em !important;
+    border-bottom: 1px solid !important;
+}
+
+form input[type="text"] {
+  border-bottom: solid 1px black !important;
+  height: 2em !important;
+  padding: 1em !important;
+}
+
+form label {
+  font-size: large;
+}
+
+.table-width {
+    max-width: 75%;    
+}
```

Comment:
Unsure if this is intentional or not, but this actually effects every page on the entire website.